### PR TITLE
(XW-1969) Use pre-compiled CSS from DS

### PR DESCRIPTION
**Pull request by Grunny**
Use pre-compiled CSS from DS to ensure we have prefixed attributes.

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [DesignSystemHooks.class.php#L14](https://github.com/Wikia/app/blob/128cba92136ef483bc6c15e8fd3a3aaba8703800/extensions/wikia/DesignSystem/DesignSystemHooks.class.php#L14): Remove the unused function parameter "$out". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [DesignSystemHooks.class.php#L14](https://github.com/Wikia/app/blob/24d05b5339e2578a759dccab084a65e536bb691d/extensions/wikia/DesignSystem/DesignSystemHooks.class.php#L14): Remove the unused function parameter "$out". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)

File: extensions/wikia/DesignSystem/DesignSystemHooks.class.php [hunk 1]
```diff
@@ -14,7 +14,7 @@ class DesignSystemHooks {
 	public static function onBeforePageDisplay( $out, $skin ) {
```
Comment (WikiaTech) [inline extensions/wikia/DesignSystem/DesignSystemHooks.class.php]: MAJOR Remove the unused function parameter "$out". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [DesignSystemHooks.class.php#L14](https://github.com/Wikia/app/blob/636f53409d3a34d2fb837c79d9da587e4bc57db1/extensions/wikia/DesignSystem/DesignSystemHooks.class.php#L14): Remove the unused function parameter "$out". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/DesignSystem/bower.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
   "private": true,
 
   "dependencies": {
-    "design-system": "Wikia/design-system#0.5.0"
+    "design-system": "Wikia/design-system#25c213c9c18b1d3c741c119ea34868f8c611c78f"
```
Comment (Grunny) [inline extensions/wikia/DesignSystem/bower.json]: Will be updated to a release version once https://github.com/Wikia/design-system/pull/54 is merged.

**Comment by kvas-damian** [comment]
Can we make sure we'll remove our hacks in: https://github.com/Wikia/app/blob/XW-1928/extensions/wikia/DesignSystem/styles/design-system.scss#L28-L94

**Comment by kvas-damian** [comment]
besides that it LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [DesignSystemHooks.class.php#L14](https://github.com/Wikia/app/blob/592c3d2832e610bc77dd9a3836f36f54d29d8621/extensions/wikia/DesignSystem/DesignSystemHooks.class.php#L14): Remove the unused function parameter "$out". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)

**Pull request merged**